packages/gatsby/src/bootstrap/index.ts

```diff
@@ -0,0 +1,50 @@
+import { startRedirectListener } from "./redirects-writer"
+import {
+  IBuildContext,
+  initialize,
+  customizeSchema,
+  sourceNodes,
+  buildSchema,
+  writeOutRequires,
+  createPages,
+  createPagesStatefully,
+  extractQueries,
+} from "../services"
+import { Runner } from "./create-graphql-runner"
+import { writeOutRedirects } from "../services/write-out-redirects"
+import { postBootstrap } from "../services/post-bootstrap"
```

Comment:
any reasons why those are imported from their files and not from `"../services"` like every other service?

Reply:
TypeScript/VS Code being weird! I'll fix

Reply:
that's minor/NIT thing, so no need to do this quickly, it just gave me a pause ("what's different about those?")

Reply:
No, it is a mistake. I mostly let it auto-import, and sometimes it does it from the wrong place